app/components/dashboard/moab_storage_roots_component.html.erb

```diff
@@ -41,13 +42,14 @@
           <td class="table-secondary"/>
           <td class="table-secondary"/>
           <td class="table-secondary"/>
-          <td class="text-end<%= ' table-danger' if storage_root_total_count != num_complete_moabs %>"><%= storage_root_total_count %></td>
-          <td class="text-end"><%= storage_root_total_ok_count %></td>
-          <% storage_root_totals[2..-2]&.each do |count| %>
-            <td class="text-end<%= ' table-danger' if count.positive? %>"><%= count %></td>
-          <% end %>
-          <% total_num_expired_checksum = storage_root_totals[7] %>
-          <td class="text-end<%= ' table-warning' if total_num_expired_checksum.positive? %>"><%= total_num_expired_checksum %></td>
+          <td class="text-end<%= ' table-danger' unless storage_roots_moab_count_ok? %>"><%= storage_roots_moab_count %></td>
```

Comment:
adding individual methods for the different total counts and for the "total count ok?" methods makes this a lot more readable too.